crates/winch/Cargo.toml

```diff
@@ -11,7 +11,7 @@ smallvec = { version = "1.6.1" }
 anyhow = "1.0"
 wasmtime-environ = { path = "../environ", version = "=2.0.0" }
 target-lexicon = "0.12"
-cranelift-codegen = { path = "../../cranelift/codegen", version = "0.89.0" }
+cranelift-codegen = { path = "../../cranelift/codegen", version = "0.89.0", features = ["x86", "arm64"] }
```

Comment:
This should probably replaced with features on winch that enable the respective feature of cranelift-codegen and additionally enabling the target architecture if no architecture is selected, just like for cranelift-codegen.